### ISSUE TITLE
Unified Pricing Model

### ISSUE THREAD

**Issue by jlogelin** [application-research/estuary-hosted-infrastructure#19]

**Comment by jlogelin** [comment]
cc: @jimmylee @jcace @alvin-reyes @schreck23

**Comment by jlogelin** [comment]
Ok, here is an explanation of the [current iteration](https://docs.google.com/spreadsheets/d/1_ZnsiL_N4s4c-_KrKE5XqsqugvkLfg-nkkqWfYN0lWc/edit#gid=245241178) of the pricing model and sheets, special thanks to 
@Zorlin for helping me get a snapshot of reality to get these models more truthy than they were. The first 4 tabs are the most significant ones, whereby the underlying costs are outlined and combined with a Model A business scenario.

**FDI TCO Infra (current)**
This is a living snapshot of our CapEx and Opex.
tldr; Our initial investment is around $300K, our monthly operating expenses run at around $3600

**FDI Financial Timeline (current)**
This is a living representation of all of our expenditures from FDI’s inception into the next forecasted 5 years. It maps out when and what we are expecting to spend our money on, and our storage capacity (raw, useable and offsite). This represents the now and the foreseeable reality.

**Model A**
This is where it gets sort of fun. Model A is a theoretical model that anyone can use to forecast a simple 5 year business model. The fields Estimated Initial Setup and Estimated Monthly Opex can be edited. I’ve seeded them with $300K and $3600 to based on what we now know.
Importantly, the parameters of the Model itself are below the estimated profit chart. All of these fields are editable and will reflect on the chart and monthly profit map above. It is noteable that, with this version of the model, the break even is $4.75 and not the original $3.00 that was originally estimated. (I dropped the ball on the previous value, as there was no accounting for staff hours... apparently people like to get paid, who knew). It’s very important to remember that this number reflects serving fast egress - that’s our special sauce, the entire experience. (ie, this is not deep glacier storage, it is instant egress)
tldr; onboarding 4 PiB/month at $100k/PiB/year is a lucrative

**FDI Timeline with w/ Model A**
This is a combined amalgamation of reality + our best guess modelling from above. It maps the profits from Model A onto the FDI Financial Timeline, giving a rough estimate of what we could expect to see if we onboard a targeted 3-4 PiB/month.
Ignore the last two sheets, they will likely go away.
Still todo, (but I would like your feedback first), is refactoring the model into a quarterly instead of monthly cadences

**Comment by jlogelin** [comment]
https://docs.google.com/spreadsheets/d/1_ZnsiL_N4s4c-_KrKE5XqsqugvkLfg-nkkqWfYN0lWc/edit#gid=360804146

**Comment by jlogelin** [comment]
Added [Model B](https://docs.google.com/spreadsheets/d/1_ZnsiL_N4s4c-_KrKE5XqsqugvkLfg-nkkqWfYN0lWc/edit#gid=1088812640) to reflect Corvault cost model
